site/docs/features/index.md

```diff
@@ -1,107 +1,327 @@
-# Introduction
+# About Nessie
 
-Nessie is an OSS service and libraries that enable you to maintain multiple versions 
-of your data and leverage Git-like Branches & Tags for your Data Lake. Nessie enhances the following 
-table formats with version control techniques:
+Nessie is to Data Lakes what git is to source code repositories. Therefore,
+Nessie uses many terms from both git and data lakes.
 
-* Apache Iceberg Tables ([more](../tables/iceberg.md))
-* Delta Lake Tables ([more](../tables/deltalake.md))
-* Hive Metastore Tables ([more](../tables/hive.md))
-* SQL Views ([more](../tables/views.md))
+This page explains how Nessie makes working with data in data lakes much easier
+without requiring much prior knowledge of either git nor data lakes.
 
-## Basic Concepts
+Nessie is designed to give users an always-consistent view to their data
+across all involved data sources (tables). Changes to your data, for example
```

Comment:
```suggestion
across all involved data sets (tables). Changes to your data, for example
```
"set" is maybe a bit more generic than "source" as it could also be a "destination"